services/ui-src/src/components/layout/Footer.tsx

```diff
@@ -9,48 +9,57 @@ import mcrLogo from "assets/logos/logo_mcr_footer.png";
 import medicaidLogo from "assets/logos/logo_medicaid.svg";
 
 export const Footer = () => {
-  const { isMobile } = useBreakpoint();
+  const { isDesktop } = useBreakpoint();
   return (
     <Box sx={sx.root} id="footer" role="contentinfo">
       <Box sx={sx.footerTop}>
         <Container sx={sx.footerTopContainer}>
           <Flex sx={sx.footerTopFlex}>
-            <Flex sx={sx.footerTopLogoFlex}>
+            <Box sx={sx.footerTopLeftContainer}>
               <Image
                 src={mcrLogo}
                 alt="Managed Care Reporting logo"
                 sx={sx.mcrLogo}
               />
-            </Flex>
-            <Flex sx={sx.footerRightLogosFlex}>
-              {!isMobile && (
-                <Image src={hhsLogo} alt="HHS logo" sx={sx.hhsLogo} />
-              )}
-              <Flex sx={sx.footerRightLogosTextFlex}>
-                {isMobile && (
-                  <Flex sx={sx.hhsMedicaidLogoMobile}>
-                    <Image src={hhsLogo} alt="HHS logo" sx={sx.hhsLogo} />
+            </Box>
+            <Box sx={sx.footerTopRightContainer}>
+              <Flex sx={sx.footerTopRightTopFlex}>
+                <Box sx={sx.footerCMSBrandingLeft}>
+                  <Box sx={sx.hhsLogo}>
+                    <img
+                      src={hhsLogo}
+                      alt="Department of Health and Human Services, USA"
+                    />
```

Comment:
Suggestion: Recommend using Chakra's `Image` component here and elsewhere in PR for consistency.